apps/cb-ceiling/test/cbQueryTests/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cbQueryTests",
+  "version": "1.0.0",
+  "description": "Couchbase query tests",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "author": "Gopa",
+  "license": "ISC",
+  "devDependencies": {
+    "jest": "^29.2.2"
+  },
+  "dependencies": {
+    "couchbase": "^4.1.3",
+    "jest": "^29.2.2"
```

Comment:
Jest is already included as a `devDependency` here so we don't need it as a production dependency.

```suggestion
```

Reply:
Done